skycam_node:
  ros__parameters:
    model_path: "/home/williamfbx/Lunar-ROADSTER/lr_ws/src/skycam/skycam_util/skycam_convnet_ts.pt"
    image_size: 224
    channels: 3
    device: "cpu"
